Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -444,10 +443,12 @@ - (MPKitExecStatus *)routeCommerceEvent:(MPCommerceEvent *)commerceEvent {
                 }
             }
             if (productArray.count > 0) {
-                properties[productKey] = productArray;
+                [properties setValue:productArray forKey:productKey];
             }
             
-            [appboyInstance logPurchase:enhancedCommerceProductId
+            NSString *eventName = [NSString stringWithFormat:@"eCommerce - %@", [self eventNameForAction:commerceEvent.action]];
```

Comment:
Is there not a name on the expanded commerce event by default from the core Apple SDK?